sympy/printing/octave.py

```diff
@@ -432,6 +435,24 @@ def _print_LambertW(self, expr):
         return "lambertw(" + args + ")"
 
 
+    def _print_math_func(self, expr):
+        known = self.known_functions[expr.__class__.__name__]
+        args = self._print(expr.args[0])
+        if len(expr.args) > 1:
+            args += ', %s' % self._print(expr.func(*expr.args[1:]))
+        
+        return '{name}({args})'.format(
+            name=known,
+            args=args
+        )
+
+    def _print_Max(self, expr):
```

Comment:
this could be written as:
`_print_Max = _print_math_func`
(same for Min)

Reply:
I didn't get this actually, @bjodah . Could you elaborate a little more?
And does this PR need any other changes?